weave

```diff
@@ -803,6 +807,8 @@ destroy_bridge() {
     run_iptables -t filter -D INPUT -i $DOCKER_BRIDGE -p udp --dst $DOCKER_BRIDGE_IP --dport $PORT          -j DROP >/dev/null 2>&1 || true
     run_iptables -t filter -D INPUT -i $DOCKER_BRIDGE -p udp --dst $DOCKER_BRIDGE_IP --dport $(($PORT + 1)) -j DROP >/dev/null 2>&1 || true
 
+    run_iptables -t filter -D FORWARD -i $BRIDGE ! -o $BRIDGE -j ACCEPT 2>/dev/null || true
+    run_iptables -t filter -D FORWARD -o weave -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT 2>/dev/null || true
```

Comment:
I am wondering why this rule has `-o weave` and not `-o $BRIDGE`? (given we set: `BRIDGE=weave` earlier)
Is it guaranteed that there will always be a bridge named `weave`? (it seems that `create_bridge` in the `weave` script always uses `$BRIDGE`, so I'm wondering if this could break if someone sets `BRIDGE` to a different value than the default)